lib/src/working_copy.rs

```diff
@@ -973,7 +973,11 @@ impl TreeState {
                     err,
                 })?;
                 let mut content = vec![];
-                file.read_to_end(&mut content).unwrap();
+                file.read_to_end(&mut content)
+                    .map_err(|err| SnapshotError::IoError {
+                        message: format!("Failed to read file {}", disk_path.display()),
+                        err,
+                    })?;
```

Comment:
nit: unrelated to this commit, but we can also use `fs::read()`

Reply:
True. I'll try to remember to change that in another PR.